src/AWS/Lambda/Runtime.hs

```diff
@@ -318,7 +321,7 @@ fallibleRuntime = ValueRuntime.fallibleRuntime . withInfallibleParse
 -- @
 pureRuntimeWithContext :: (FromJSON event, ToJSON result) =>
   (LambdaContext -> event -> result) -> IO ()
-pureRuntimeWithContext = ValueRuntime.pureRuntimeWithContext . fmap withInfallibleParse
+pureRuntimeWithContext fn = ValueRuntime.fallibleRuntimeWithContext $ withFallibleParse . fn
```

Comment:
If we aren't using the same level in the hierarchy, I think it makes sense to stay within this current module.  Couldn't this be `fallibleRuntimeWithContext . fmap (fmap pure)`?

Reply:
Yes that's true. Thanks for this suggestion.